tools/bwa.py

```diff
@@ -52,7 +52,7 @@ def index(self, inFasta, output=None, algorithm=None):
         self.execute('index', cmd)
 
     def align_mem_bam(self, inBam, refDb, outBam, options=None,
-                      min_score_to_filter=None, threads=None, JVMmemory=None):
+                      min_score_to_filter=None, threads=None, JVMmemory=None, invert_filter=False):
```

Comment:
The `invert_filter` argument is for the future (speculative), should we desire to do depletion based on alignment score rather than the `-f0x4` flag.